docs/separate-benchmark-module.md

```diff
@@ -0,0 +1,102 @@
+# Creating Separate Module for Benchmarks
+
+This guide will walk you through the process of creating a separate module for benchmarks in your Kotlin project. 
+Creating a separate module for benchmarks is especially beneficial when you are integrating benchmarks into an existing project. 
+Here are a few advantages of doing so:
+
+1. **Organization**: It helps maintain a clean and organized project structure. Segregating benchmarks from the main code makes it easier to navigate and locate specific code segments.
```

Comment:
But to some degree that could be also achieved by using a separate source set, isn't it?

Reply:
It can be. However, in K/Multiplatform projects each source set has to be included into a compilation, thus there is no way to avoid compilations.
Including the source set with benchmarks into the main compilation means:
* The benchmark code is also published together with the main code.
* Dependencies of the benchmark code will be in the list of dependencies of the published artifact.

Perhaps it's possible to avoid these problems, but it needs more advanced configurations of the project.

The recommended (by the library) way is to set up the benchmark compilation similar to the test compilation, which is separated from the main compilation.